AzFilesHybrid/AzFilesHybridUtilities.ps1

```diff
@@ -0,0 +1,30 @@
+function Write-FailedPSStyle(
+    [Parameter(Mandatory=$true, Position=0)]
+    [string]$message
+) {
+    [string]$redFailed = "`t$($PSStyle.Foreground.BrightRed)ERROR$($PSStyle.Reset)"
+    Write-Host "${redFailed}: $message"
+}
+
+function Write-WarningPSStyle(
+    [Parameter(Mandatory=$true, Position=0)]
+    [string]$message
+) {
+    [string]$warning = "`t$($PSStyle.Foreground.BrightYellow)WARNING$($PSStyle.Reset)"
+    Write-Host "${warning}: $message"
+}
+
+function Write-TestingPassedPSStyle() {
+    $checkmark = [System.Char]::ConvertFromUtf32([System.Convert]::ToInt32("2713", 16))
+    Write-Host "`r$($PSStyle.Foreground.BrightGreen)`t`t($checkmark) Passed$($PSStyle.Reset)"
```

Comment:
Maybe a single tab is enough here. Or maybe we can make it configurable like the suggestion above.

Reply:
I played around with testing the carriage return (`r), and it looks like indentation (`t) sets where the cursor goes back to in the output. It's definitely something that should be configurable like you suggested! I pasted how indentation affects the carriage return below for reference:
No indentation
![image](https://github.com/user-attachments/assets/543c8f1c-b5c5-4efa-bdbc-49124266d3b2)

Single indentation (`t)
![image](https://github.com/user-attachments/assets/cd6fb8d5-2646-4857-baff-816e992f604e)

Double indentation (`t`t)
![image](https://github.com/user-attachments/assets/ba76b8ca-e50f-44c5-895f-034f52ba49c5)